tests/chainer_tests/test_version.py

```diff
@@ -0,0 +1,10 @@
+import unittest
+import packaging.version
```

Comment:
Could you group imports?  `packaging` is not a built-in module.
```suggestion

import packaging.version
```